Experimental/DistributedAuthoritySample/Assets/Scripts/Player/AvatarTransform.cs

```diff
@@ -47,30 +47,63 @@ public override void OnNetworkSpawn()
 
             this.RegisterNetworkUpdate(updateStage: NetworkUpdateStage.Update);
             this.RegisterNetworkUpdate(updateStage: NetworkUpdateStage.FixedUpdate);
+
+            var cameraControl = Camera.main?.GetComponent<CameraControl>();
+            if (cameraControl != null)
+            {
+                cameraControl.SetTransform(transform);
+                SetCameraReference(Camera.main);
```

Comment:
Since you're already grabbing the camera, you don't need to call SetCameraReference externally from CameraControl. You could just replace line 55 with m_MainCamera = Camera.main.

Reply:
Oooh you are right, thank you!